tests/Activities.UnitTests/Internal/ActivityMetricsSenderTests.cs

```diff
@@ -17,7 +19,70 @@ namespace Microsoft.Omex.Extensions.Activities.UnitTests.Internal
 	public class ActivityMetricsSenderTests
 	{
 		[TestMethod]
-		public void SendActivityMetric_ProduceMetrics()
+		[DataRow("")]
+		[DataRow(null!)]
+		public void ActivityMetricsSender_Construction_NonExistConfigSection_ReturnNotNull(string nonExistValue)
+		{
+			Mock<IExecutionContext> contextMock = new();
+			Mock<IHostEnvironment> environmentMock = new();
+			Mock<IConfiguration> mockConfig = new();
+			Mock<IConfigurationSection> mockConfigSection = new();
+
+			mockConfig.Setup(configuration => configuration.GetSection(ActivityMetricsSender.UseHistogramForActivityMonitoringConfigurationPath))
+				.Returns(mockConfigSection.Object);
```

Comment:
nit: I would chain the extension method calls vertically to increase readability (in the specific case, put `.GetSection` call one line below, indented with `.Returns`, and then leave a line below the `.Returns`.